tests/core/test_core_cli.py

```diff
@@ -457,54 +456,42 @@ def test_force_override_production_version():
         core.stop()
 
 
-@pytest.mark.standalone
 def test_modified_job_configuration_dont_block_application_run(caplog, init_config):
-    scenario_config = config_scenario()
+    _ = config_scenario()
```

Comment:
I think we can just keep it even simpler with 
```suggestion
    config_scenario()
```

Reply:
this will trigger a ruff error where the return of `config_sceario()` is not assigned to any variable

Reply:
I see :D that's ok to me too